Patches/PsiTech/PawnKindDefs/PawnKinds_PsiTech.xml

```diff
@@ -0,0 +1,215 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>PsiTech</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+				<!-- ==== Psion_Penitent ==== -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/PawnKindDef[defName="Psion_Penitent"]/combatPower</xpath>
+					<value>
+						<combatPower>20</combatPower>
```

Comment:
Some of these combatPower values strike me as being fairly low (for reference, a tribal warrior = 24, tribal archer = 30). Can you do some testing with some raids using the point values to ensure they're resulting in raid sizes that are reasonable? Being attacked by 10 guys with guns (even crappy ones) is a lot different than being attacked by 10 guys with melee weapons.

Reply:
I'll do some test runs of me trying to fend off same point raids from the modded and vanilla pirate factions, then see how much do `combatPower` values need tweaking.

Reply:
I did some tests and tweaking. Would be good if you would comment on the changes.

Reply:
Seems to be good.